os/include/tinyara/binary_manager.h

```diff
@@ -70,18 +70,38 @@
 /* Mount point for User binaries */
 #define BINARY_DIR_PATH                  CONFIG_MOUNT_POINT"bins"
 
+/* The length of binary file or kernel partition path.
+ * A path is same like below:
+ *  - User filepath : "/<mount_path>/<binary_name>_<binary_version>"
+ *         length : CONFIG_NAME_MAX + BIN_NAME_MAX + binary_version (32) + /,_(3) + '\0'(1)
+ *  - Kernel partpath : "/dev/mtdblock#" defined as BINMGR_DEVNAME_FMT
+ * So define binary path length as (CONFIG_NAME_MAX + BIN_NAME_MAX + 40) with some extra.
+ */
+#define BINARY_PATH_LEN                  (CONFIG_NAME_MAX + BIN_NAME_MAX + 40)
```

Comment:
What is the meaning of 40?

Reply:
The maximum length of user file path is 36 and I added some extra.